### PR TITLE
Try to remove `skip: true  # [win]`

### PR DESCRIPTION
**Pull request by paugier**
Try to build on Windows.

Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by paugier** [comment]
@conda-forge-admin, please rerender

**Comment by paugier** [comment]
@CyrilleBonamy It would be great if you can have a look at this. 

We may have to add something like `skip: true  # [win and py27]` but it is important to get Pythran conda packages for Windows and Python >= 3.6.

**Comment by paugier** [comment]
@conda-forge/staged-recipes Any ideas how we can debug this problem? The error log is very short...

It should at least work for Python 3.6 and 3.7.

CC @serge-sans-paille

**Comment by saraedum** [comment]
@paugier: Have you tried asking on https://gitter.im/conda-forge/conda-forge.github.io?

**Comment by ocefpaf** [comment]
The error message is:

```
WARNING:conda_build.build:failed to get install actions, retrying.  exception was: Unsatisfiable dependencies for platform win-64: {'pythran==0.9.1post0=py27_blas_openblashdc96acc_1'}
```

that means a dependencies required to build `pythran` does not exist on Windows. My guess is that it is `gmp`.

**Comment by serge-sans-paille** [comment]
And gmp is no longer a pythran deps, so we should update the whole recipe :-)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,9 +13,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   script: python -m pip install --no-deps --ignore-installed .
-  skip: true  # [win]
   features:
     - blas_{{ variant }}
```
Comment (isuruf) [inline recipe/meta.yaml]: Remove this line. It's not needed.

File: .ci_support/win_cxx_compilervs2015python3.7.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+channel_sources:
+- conda-forge,defaults
+channel_targets:
+- conda-forge main
+cxx_compiler:
+- vs2015
+gmp:
```
Comment (serge-sans-paille) [inline .ci_support/win_cxx_compilervs2015python3.7.yaml]: There's still a reference to gmp here

File: .ci_support/win_cxx_compilervs2015python3.6.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+channel_sources:
+- conda-forge,defaults
+channel_targets:
+- conda-forge main
+cxx_compiler:
+- vs2015
+gmp:
```
Comment (serge-sans-paille) [inline .ci_support/win_cxx_compilervs2015python3.6.yaml]: There's still a reference to gmp here

File: .ci_support/win_cxx_compilervs2008python2.7.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+channel_sources:
+- conda-forge,defaults
+channel_targets:
+- conda-forge main
+cxx_compiler:
+- vs2008
+gmp:
+- '6'
+pin_run_as_build:
+  gmp:
```
Comment (serge-sans-paille) [inline .ci_support/win_cxx_compilervs2008python2.7.yaml]: There's still a reference to gmp here

File: .ci_support/osx_python3.7.yaml [hunk 5]
```diff
@@ -6,6 +6,8 @@ channel_targets:
 - conda-forge main
 cxx_compiler:
 - clangxx
+cxx_compiler_version:
+- '4'
 gmp:
```
Comment (serge-sans-paille) [inline .ci_support/osx_python3.7.yaml]: There's still a reference to gmp here

File: .ci_support/osx_python3.6.yaml [hunk 6]
```diff
@@ -6,6 +6,8 @@ channel_targets:
 - conda-forge main
 cxx_compiler:
 - clangxx
+cxx_compiler_version:
+- '4'
```
Comment (serge-sans-paille) [inline .ci_support/osx_python3.6.yaml]: There's still a reference to gmp here

File: .ci_support/osx_python2.7.yaml [hunk 7]
```diff
@@ -6,6 +6,8 @@ channel_targets:
 - conda-forge main
 cxx_compiler:
 - clangxx
+cxx_compiler_version:
+- '4'
 gmp:
```
Comment (serge-sans-paille) [inline .ci_support/osx_python2.7.yaml]: There's still a reference to gmp here

File: .ci_support/linux_python3.7.yaml [hunk 8]
```diff
@@ -4,6 +4,8 @@ channel_targets:
 - conda-forge main
 cxx_compiler:
 - gxx
+cxx_compiler_version:
+- '7'
 docker_image:
 - condaforge/linux-anvil-comp7
```
Comment (serge-sans-paille) [inline .ci_support/linux_python3.7.yaml]: There's still a reference to gmp here

File: .ci_support/linux_python3.6.yaml [hunk 9]
```diff
@@ -4,6 +4,8 @@ channel_targets:
 - conda-forge main
 cxx_compiler:
 - gxx
+cxx_compiler_version:
+- '7'
 docker_image:
 - condaforge/linux-anvil-comp7
```
Comment (serge-sans-paille) [inline .ci_support/linux_python3.6.yaml]: There's still a reference to gmp here

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Pull request merged**